aspnetcore/security/enforcing-ssl.md

```diff
@@ -343,38 +343,75 @@ The following command provides help on the `dev-certs` tool:
+
+### Firefox SEC_ERROR_INADEQUATE_KEY_USAGE certificate error
+
+The Firefox browser uses it's own certificate store, and therefore doesn't trust the [IIS Express](/iis/extensions/introduction-to-iis-express/iis-express-overview) or [Kestrel](xref:fundamentals/servers/kestrel) developer certificates.
+
+To use Firefox with IIS Express or Kestrel, set  `security.enterprise_roots.enabled` = `true`
+
+1. Enter `about:config` in the FireFox browser.
+1. Select **Accept the Risk and Continue** if you accept the risk.
+1. Select **Show All**
+1. Set `security.enterprise_roots.enabled` = `true`
+1. Exit and restart Firefox
+
+For more information, see [Setting Up Certificate Authorities (CAs) in Firefox](https://support.mozilla.org/kb/setting-certificate-authorities-firefox) and the [`mozilla/policy-templates/README file](https://github.com/mozilla/policy-templates/blob/master/README.md).
+
 ## How to set up a developer certificate for Docker
 
 See [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/6199).
 
 <a name="ssl-linux"></a>
 
-## Trust HTTPS certificate on Linux
+## ~Trust HTTPS certificate on Linux~
 
 <!-- Instructions to be updated by engineering team after 5.0 RTM. -->
 
-For instructions on Linux, refer to the distribution documentation.
+~For instructions on Linux, refer to the distribution documentation.~
 
 <a name="wsl"></a>
 
 ## Trust HTTPS certificate from Windows Subsystem for Linux
 
-The Windows Subsystem for Linux (WSL) generates an HTTPS self-signed cert. To configure the Windows certificate store to trust the WSL certificate:
+The [Windows Subsystem for Linux (WSL)](/windows/wsl/about) generates an HTTPS self-signed cert. To configure the Windows certificate store to trust the WSL certificate:
 
-* Run the following command to export the WSL-generated certificate:
+* Export the developer certificate to a file on ***Windows***
 
   ```
-  dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <cryptic-password>
+  dotnet dev-certs https --ep C:\<<path-to-folder>>\aspnetcore.pfx -p $CREDENTIAL_PLACEHOLDER$
   ```
-* In a WSL window, run the following command:
+  Where `$CREDENTIAL_PLACEHOLDER$` is a password.
+
+* In a WSL window, import the exported certificate on the WSL instance:
 
   ```
-    ASPNETCORE_Kestrel__Certificates__Default__Password="<cryptic-password>" 
-    ASPNETCORE_Kestrel__Certificates__Default__Path=/mnt/c/Users/user-name/.aspnet/https/aspnetapp.pfx
-    dotnet watch run
+  dotnet dev-certs https --clean --import /mnt/c/<<path-to-folder>>/aspnetcore.pfx -p <<password>>
   ```
 
-  The preceding command sets the environment variables so Linux uses the Windows trusted certificate.
+  The preceding approach is a one time operation per certificate and per WSL distribution. It's easier than exporting the certificate over and over. If you update or regenerate the certificate on windows, you might need to run the preceding commands again.
```

Comment:
> It's easier than exporting the certificate over and over.

When would you need to do it over? Can't we remove this. 
***is a one time operation per certificate and per WSL distribution.*** should be all we need to say